tensorflow/tools/tf_windows_build_dockerfiles/README.md

```diff
@@ -0,0 +1,23 @@
+**To build a Windows container locally, execute these commands. These commands have been tested on Windows PowerShell when run as an administrator:**
```

Comment:
I'm not sure this is needed?

Reply:
Yes, but we don't need them in the repo, they can be just a comment in the PR

Reply:
Done